server.js

```diff
@@ -1,11 +1,12 @@
 /**
  * Module dependencies.
  */
-var express = require('express'),
-    fs = require('fs'),
-    passport = require('passport'),
-    logger = require('mean-logger'),
-    io = require('socket.io');
+const express = require('express'),
+  fs = require('fs'),
+  passport = require('passport'),
+  logger = require('mean-logger'),
+  io = require('socket.io'),
+  dotenv = require('dotenv').config();
```

Comment:
Please review the hound bot on your code, fix the merge conflicts, and raise another PR